docs/components/gripper/_index.md

```diff
@@ -325,14 +325,14 @@ if geometries:
 For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/resource#Shaped).
 
 ```go {class="line-numbers linkable-line-numbers"}
-myGripper, err := gripper.FromRobot(robot, "my_gripper")
+myGripper, err := gripper.FromRobot(machine, "my_gripper")
 
 geometries, err := myGripper.Geometries(context.Background(), nil)
 
 if len(geometries) > 0 {
     // Get the center of the first geometry
     elem := geometries[0]
-    fmt.Println("Pose of the first geometry's center point:", elem.center)
+    fmt.Println("Pose of the first geometry's center point:", elem.Pose())
```

Comment:
for the center point, it looks like `Pose` is the right choice. in `spatialmath/geometry.go` it says `// Pose returns the Pose of the center of the Geometry`

as for the errors, it looks like geometries should still exist in the Go SDK so I'm not sure why you're getting these errors. it might be specific to the type of gripper you're using?